### PR TITLE
[MIG][10.0-Issue175-T23299] auth_ldap_self_signed

### PR DESCRIPTION
**Pull request by sudhir-serpentcs**
Migrated auth_ldap_self_signed in v10.
@elicoidal @seb-elico @victormartinelicocorp Please review.
This PR refers to issue #175

### CODE REVIEW

File: auth_ldap_self_signed/README.rst [hunk 1]
```diff
@@ -1,4 +1,4 @@
-.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+ .. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```
Comment (elicoidal) [inline auth_ldap_self_signed/README.rst]: remove starting space

File: auth_ldap_self_signed/README.rst [hunk 2]
```diff
@@ -27,16 +25,13 @@ For further information, please visit:
...
-======================
-
 Bug Tracker
 ===========
 
 Bugs are tracked on `GitHub Issues <https://github.com/Elico-Corp/odoo/issues>`_.
 In case of trouble, please check there if your issue has already been reported.
 If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
-`here <https://github.com/Elico-Corp/odoo/issues/new?body=module:%20display_db_id%0Aversion:%20{8.0}%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+`here <https://github.com/Elico-Corp/odoo-addons/issues/new?body=module:%20display_db_id%0Aversion:%20{10.0}%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
Comment (elicoidal) [inline auth_ldap_self_signed/README.rst]: actually remove this line in new template

File: auth_ldap_self_signed/README.rst [hunk 3]
```diff
@@ -27,16 +25,13 @@ For further information, please visit:
 
  * https://www.odoo.com/forum/help-1
 
-Known issues / Roadmap
-======================
-
 Bug Tracker
 ===========
 
 Bugs are tracked on `GitHub Issues <https://github.com/Elico-Corp/odoo/issues>`_.
```
Comment (elicoidal) [inline auth_ldap_self_signed/README.rst]: update to https://github.com/Elico-Corp/odoo-addons/issues

File: auth_ldap_self_signed/models/__init__.py [hunk 4]
```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# © 2015 Elico corp (www.elico-corp.com)
```
Comment (elicoidal) [inline auth_ldap_self_signed/models/__init__.py]: add 2017

**Review by elicoidal — CHANGES_REQUESTED** [review]
small details. LGTM

**Comment by elicoidal** [comment]
Can you check travis?

**Comment by sudhir-serpentcs** [comment]
Improved the content of README and copyrights as per GIT review.
Travis is red due to "report_py3o" as it is not able to find the OCA module.
@elicoidal @seb-elico @victormartinelicocorp Please review.

File: .travis.yml [hunk 5]
```diff
@@ -32,10 +32,15 @@ env:
   - VERSION="10.0" TESTS="0" LINT_CHECK="0" TRANSIFEX="0"
 
 install:
+  - pip install ezodf==0.3.2
```
Comment (martintorresvm) [inline .travis.yml]: @sudhir-serpentcs To modify travis better on a new PR. 

@seb-elico ?

File: oca_dependencies.txt [hunk 6]
```diff
@@ -0,0 +1,3 @@
+# List the OCA project dependencies, one per line
+# Add a repository url and branch if you need a forked version
+reporting-engine https://github.com/OCA/reporting-engine.git 10.0
```
Comment (elicoidal) [inline oca_dependencies.txt]: do we need this for this module?

**Review by elicoidal — COMMENTED** [review]
@seb-elico can you check Travis?

File: .travis.yml (hunk 5 above)
Comment (sudhir-serpentcs) [reply .travis.yml]: Oh, yes. My intention was just to make travis green.
@victormartinelicocorp @seb-elico

File: oca_dependencies.txt (hunk 6 above)
Comment (sudhir-serpentcs) [reply oca_dependencies.txt]: @elicoidal Yes, we will need this as 10.0 branch has some modules that depends on "report_py3o" module.
@victormartinelicocorp @seb-elico

Comment (elicoidal) [reply oca_dependencies.txt]: ok

File: .travis.yml (hunk 5 above)
Comment (elicoidal) [reply .travis.yml]: It is ok: Let's fix everything on this PR and move quick

File: auth_ldap_self_signed/__init__.py [hunk 7]
```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-# © 2015 Elico corp (www.elico-corp.com)
```
Comment (elicoidal) [inline auth_ldap_self_signed/__init__.py:2]: both 2015-2017 (everywhere)
In all PR, please check: keep the original date and add the new one.

**Review by elicoidal — COMMENTED** [review]
small detaill. LGTM
@seb-elico Either we have a runbot or we quickly need to mount a trunk environment pointing to the different PR

**Comment by victor-ditay** [comment]
@sudhir-serpentcs please have a check on latest comments from @elicoidal others LGTM

**Comment by sudhir-serpentcs** [comment]
@victormartinelicocorp Done

**Comment by iceyxu** [comment]
@Elico-Corp/odoo-reviewers what status for this PR? Can we process to next step?

**Review by elicoidal — APPROVED** [review]

**Comment by elicoidal** [comment]
To be tested

**Pull request merged**